### PR TITLE
minor file type

### PR DESCRIPTION
**Pull request by kordjamshidi**

### CODE REVIEW

File: saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/SpRLApp.scala [hunk 1]
```diff
@@ -55,7 +55,7 @@ object SpRLApp extends App with Logging {
       println("input directory doesn't exists or is empty.")
       return
     }
-    val documentList = inputDir.listFiles.map(file => {
+    val documentList = inputDir.listFiles.filter(f => f.getName.endsWith(".txt")).map(file => {
       val text = scala.io.Source.fromFile(file).mkString
```
Comment (kordjamshidi) [inline saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/SpRLApp.scala:59]: I just changed this, I have no idea what are the other changes!!

Comment (Rahgooy) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/SpRLApp.scala:59]: we don't need other changes, I guess.

Comment (kordjamshidi) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/SpRLApp.scala:59]: are those yours? or come from Umar PR?

Comment (Rahgooy) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/SpRLApp.scala:59]: part of it is old code that we have changed and other is from Umar.

File: saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLDataModel.scala [hunk 2]
```diff
@@ -288,6 +288,33 @@ object MultiModalSpRLDataModel extends DataModel {
...
+        }
+        if (sc.nonEmpty) {
+          secondConcept = sc.get.getSegmentConcept()
+        }
+        val relation = ir.getRelation()
+        val result = (x.getText().toLowerCase().contains(firstConcept.toString().toLowerCase()) || fc.get.isexistOntologyConcepts(x.getText().toLowerCase()) || x.getText().toLowerCase().contains(secondConcept.toString().toLowerCase()) || sc.get.isexistOntologyConcepts(x.getText().toLowerCase())) && (rel.getText().toLowerCase().contains(relation.toLowerCase()))
+        result
+      })
+  }
+
```
Comment (kordjamshidi) [inline saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLDataModel.scala:317]: this is not needed?

Comment (Rahgooy) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLDataModel.scala:317]: no, you can revert everything to master and make your change.

File: saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLApp.scala [hunk 3]
```diff
@@ -102,7 +102,7 @@ object MultiModalSpRLApp extends App with Logging {
         classifier.load()
         if (classifier == TrajectorPairClassifier || classifier == LandmarkPairClassifier) {
           if (!pairsPopulated) {
-            populatePairDataFromAnnotatedCorpus(xmlReader, isTrain, x => IndicatorRoleClassifier(x) == "Indicator")
+            populatePairDataFromAnnotatedCorpus(xmlReader, isTrain, x => SentenceLevelConstraintClassifiers.IndicatorConstraintClassifier(x) == "Indicator")
```
Comment (kordjamshidi) [inline saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLApp.scala:105]: and you don't use the constrained classifier for the sp classification?

Comment (kordjamshidi) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLApp.scala:105]: sorry, I am confused, I always pulled the last changes, I can not see why my PR is not consistent.

Comment (Rahgooy) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLApp.scala:105]: if you remember when we used the train/test setting it wasn't better than unconstrained one, so we decided to use that.

File: saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLDataModel.scala (hunk 2 above)
Comment (kordjamshidi) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLDataModel.scala:317]: I think I merged with you master, let me check again...

**Pull request closed without merging**